index.html

```diff
@@ -5,8 +5,152 @@
+
+        <div class="main-header-logo">
+          <a href="index.html" title="Главная страница Техномарт">
+            <img src="#" width="" height="">
+          </a>
+        </div>
+        <form action="#" method="POST"><!-- progressive enhancement -->
+          <input type="search" placeholder="Поиск:">
+        </form>
+
+        <ul class="header-menu">
+          <li><a href="#"></a>Закладки</li>
+          <li><a href="#"></a>Корзина</li>
+          <li><a href="#"></a>Оформить заказ</li>
+        </ul>
+
+      </nav>
+
+      <div class="main-description">
+        <h1>Интернет-магазин строительных материалов и инструментов для ремонта</h1>
+        <!-- ^^^ Наверное это тег i, но мне кажется он тянет
+        на заголовок первого уровня не меньше
+        -->
+      </div>
+
+      <div class="header-contacts">
+        <span class="header-phone">
+          <a href="tel:+7812550555">+ 7 (812) 55-05-55</a>
+        </span>
+        <span class="header-address">г. Санкт-Петербург, ул. Б. Конюшеннаяб д. 19/8</span>
+      </div>
+
+      <ul class="header-menu-bar">
+        <li>
+          <a class="signin" href="#">Войти</a>
+        </li>
+        <li>
+          <a class="registration" href="#">Регистрация</a>
+        </li>
+      </ul>
+
+      <nav class="main-header-nav">
+        <ul>
+          <li><a href="#">Главная</a></li>
+          <li><a href="#">Компания</a></li>
+          <li><a href="catalog.html">Каталог</a></li>
+          <li><a href="#">Новости</a></li>
+          <li><a href="#">Спецпредложения</a></li>
+          <li><a href="#">Доставка</a></li>
+          <li><a href="#">Контакты</a></li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <!-- MAIN -->
+    <main>
+      <section class="about-us">
+        <h2>Компания «Техномарт»</h2>
+        <p>Настоящий мужской шоппинг начинается здесь! Огромный выбор товаров для
+          ремонта и строительства не оставит равнодушными любителей сэкономить на
+          гастарбайтерах.
+        </p>
+        <p>Мы можем доставить ваш товар в самые отдаленные точки России!
+          Техномарт работает со мноигми транспортными компаниями:
+        </p>
+        <ul class="clients-list">
+          <li>Деловые линии</li>
+          <li>Автотрейдинг</li>
+          <li>ЖелДорЭкспедиция</li>
+        </ul>
+
+        <a href="#">Подробнее о компании</a>
+        </section>
+
+      <section class="contacts">
+        <h2>Контакты</h2>
+        <p>Вы можете забрать товар сами,
+          заехав в наш офис:
+        </p>
+        <img class="contacts-map" src="#" alt="Наш адрес: ул. Б. Конюшенная, 19/8">
+        <a href="#">Заблудились? Напиши нам!</a>
+      </section>
+
+    </main>
+
+    <!-- FOOTER -->
+    <footer class="main-footer">
+      <div class="footer-top">
+        <div class="main-footer-logo">
+          <a href="index.html" title="Главная страница Техномарт">
+            <img src="#" width="" height="">
+          </a>
```

Comment:
title хорошо, но внутри всеравно должен быть текст для скрин ридера, чтобы он зачитался.
Мы его потом добавим и скроем через специальный класс - это так на будущее

Reply:
Нашел статью в академии. Если правильно понял, то на текущий момент (тот уровень знаний, который нам уже дали) по идеи этого решения хватит.

```suggestion
          <a href="index.html" title="Главная страница Техномарт">
           <h2 hidden>Главная страница Техномарт</h2>
            <img src="#" width="" height="">
          </a>
```